### DISCUSSION TITLE
[Meta] The number of off topic discussions is getting silly

### DISCUSSION THREAD

**Discussion by Daeraxa** [community/community#59083] [Discussions]
### Select Topic Area

General

### Body

I'm trying to report them as I see them but honestly the sheer number of topics show in `GitHub Product Categories > Discussion` is just ridiculous.

Currently of the 24 topics on page 1 (excluding pinned), 14 of them aren't related to the topic.

This is despite the *two* sections explaining what this section is for - clearly something needs to be changed here. I would check this forum more regularly but I can't be bothered because there is about an 80% chance of any new topic just being somebody complaining that their student credentials haven't been accepted or "how 2 clone repo plz".

**Comment by Daeraxa** [comment]
As an update on how well this seems to be going - as of right now of the 25 topics on the front page 21 of them are nothing to do with GitHub discussions.

**Reply by karasowles** [reply]
Sorry @Daeraxa, we'll try to keep closer tabs on it. The naming of Discussions-the-feature within a place for posting Discussions can be pretty confusing to new posters choosing a topic 😮‍💨

**Reply by Daeraxa** [reply]
I certainly understand the difficulty, what has already been done is exactly what I would have tried but people still just blatantly ignore it.

I think possibly the weakest point here is the new topic page - https://github.com/orgs/community/discussions/new/choose. When you create a discussion from the generic Discussions main page you are presented with the below view. Problems I see here are:
![image](https://github.com/community/community/assets/58074586/2283d95e-033f-4da9-bd0e-12a87dba3d1e)

1) This view doesn't reflect the organisation of categories as shown in the sidebar which clearly places Discussions under the "GitHub Products" section:
![image](https://github.com/community/community/assets/58074586/6acef88e-8a88-46d2-92be-06325f7e57b2)
Instead these are organised purely alphabetically. As a general product suggestion should this not reflect the organisation put into the categories and sections?  
For example would this not be improved if, like the sidebar, instead of each category having its own "Get Started" in a flat A-Z structure, actually reflect the sections and collapse/expand them as required.  
I believe the main problem here is that "Discussions" turns up before "General" in the list. If instead it had "GitHub Products" with "Discussions" as a child item then people will be far less likely to pick the wrong category.
2) The description of the category itself is a little vague. It starts with:

> Conversations solely for GitHub Discussions-related conversations. 

Which doesn't really get across that it is for this exact product/feature.  
I have no good explanation as to why the warning in the second part doesn't catch people's attention though as that clearly explains what the category is and what it isn't. Maybe the ending line of "Please use 'General' for other misc topics." could be moved up to the top with the first line?

**Reply by karasowles** [reply]
Slapping a 'product feedback' label on this, because this is great feedback.
